src/rust/src/x509/verify.rs

```diff
@@ -73,6 +74,22 @@ pub(crate) struct PolicyBuilder {
     time: Option<asn1::DateTime>,
     store: Option<pyo3::Py<PyStore>>,
     max_chain_depth: Option<u8>,
+    ca_ext_policy: Option<ExtensionPolicy<PyCryptoOps>>,
+    ee_ext_policy: Option<ExtensionPolicy<PyCryptoOps>>,
+}
+
+impl PolicyBuilder {
+    /// Clones the builder, requires the GIL token to increase
+    /// reference count for `self.store`.
```

Comment:
In general, these sort of comments aren't needed.

Reply:
Understood, I'll remove the comment. I added it mostly to signify why this differently named method is added instead of implementing Clone, but I guess it's obvious.